src/watchful/client.py

```diff
@@ -1008,9 +1008,11 @@ def hint(name: str, offset: int, values: List[bool]) -> Optional[Dict]:
     TODO: Come up with a better streaming Python API here.
     """
 
-    values = list(map(lambda x: x, values))
+    cast_values = list(map(lambda x: int(x), values))
 
-    return _assert_success(api("hint", name=name, offset=offset, values=values))
+    return _assert_success(
+        api("hint", name=name, offset=offset, values=cast_values)
+    )
```

Comment:
Per @queenieho, "hint" isn't used in the browser. Can we investigate whether we can just drop "hint" entirely? I think it's dangerous for our app to have clients that can go down two different codepaths to accomplish the same thing.

Reply:
I'm quite happy to do that (investigate if we can/should drop `hint`) in a follow-on PR later today if you're open to that.

Reply:
I think we might have more than one endpoint like that...